README.md

```diff
@@ -1,10 +1,27 @@
 # Open Liberty Starter
 
-## How to deliver code and test
+## How to run locally in dev mode
+1) Clone the repository
+2) Run `cd start.openliberty.io`
+3) Run `mvn liberty:dev` 
+4) Then you can start testing all endpoints at `https://localhost:9443/api/`   
+
+## How to debug using IntelliJ
```

Comment:
We can reference the use of Liberty Tools and what it provides for debugging?  Cc @TrevCraw @aparnamichael

Reply:
not sure if it's only me, the Liberty Toools doesn't work for me, I get this message:
`No Liberty Maven or Liberty Gradle projects detected in this workspace.`